src/webgpu/shader/README.md

```diff
@@ -0,0 +1,3 @@
+Tests for full coverage of the shaders that can be passed to WebGPU.
+
+This folder contains WGSL auto-generated test plans. Please read the [guidelines](docs/plan_autogen.md) before contributing.
```

Comment:
FYI `README.md` files won't be picked up by the standalone runner / test plan view "description" boxes, it only looks at `README.txt` files.

It's nice for the toplevel directory to be described there, and although it's not critical I think it would be best to keep this as a .txt.

(Side note: the link as written is relative to the current directory so it wouldn't work)

Reply:
I don't think this is a useful message to be picked up be the standalone runner. I revert in this CL and will add it as a comment to the files with auto-generated test plans.